DESCRIPTION

```diff
@@ -1,12 +1,16 @@
-Package: strucchange
-Version: 1.5-9000
-Date: 2021-02-27
-Title: Testing, Monitoring, and Dating Structural Changes
-Authors@R: c(person(given = "Achim", family = "Zeileis", role = c("aut", "cre"), email = "Achim.Zeileis@R-project.org",
+Package: strucchangeRcpp
+Version: 1.5-3-1.0.0
```

Comment:
Wondering whether CRAN will accept such a version number. If not, maybe it would be enough to refer to the version of the `strucchange` fork in `NEWS.md`.

Reply:
Yes, I discussed that with @mirt001 in the last hackathon, and using this allows for easy transition for any other package that may want to use it, even though it's highly unusual :) R can handle it fine, but indeed, let's see how it goes. It's pretty easy to change if someone finds it strange.